fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBReturnedRecordLimitQueryTest.java

```diff
@@ -65,7 +67,7 @@
      * Verify that a returned record limit works properly against a query with a filter on one field and a sort on another,
      * when the filter field is un-indexed and the sort is in reverse order.
      */
-    @DualPlannerTest
+    @Test
```

Comment:
Had to remove the dual planner annotation as the temp planner cannot plan this anymore.